client/asset/doge/doge.go

```diff
@@ -154,6 +154,7 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network)
 		Segwit:                   false,
 		OmitAddressType:          true,
 		LegacySignTxRPC:          true,
+		LegacySentRawTx:          true,
```

Comment:
:+1: confirmed that dogecoind is legacy w.r.t. this:

`sendtoaddress "address" amount ( "comment" "comment_to" subtractfeefromamount )`